### PR TITLE
Updates for SDK v5.1

### PR DESCRIPTION
**Pull request by Mriegel**

### CODE REVIEW

**Comment by kbataran** [comment]
can you update the README to include cordova installation instructions/developer setup? Similar to https://github.com/localytics/react-native-template-app

**Comment by kbataran** [comment]
Probably as simple as saying `$ sudo npm install -g cordova` and linking to https://cordova.apache.org/docs/en/latest/guide/cli/

File: platforms/android/res/values/localytics.xml
Comment (kbataran) [inline platforms/android/res/values/localytics.xml:42]: This is missing the location update settings (prob need to grab the latest localytics.xml)

File: plugins/com.localytics.phonegap.LocalyticsPlugin/README.md
Comment (kbataran) [inline plugins/com.localytics.phonegap.LocalyticsPlugin/README.md:459]: another good example of why this should prob be ignored

**Review by kbataran — COMMENTED** [review]
1) Should we just integrate FCM as the standard in these template apps?
2) Should the "plugins" folder be .gitignored similar to how the "node_modules" folder is for React Native apps?

File: platforms/android/res/values/localytics.xml
Comment (Mriegel) [reply platforms/android/res/values/localytics.xml:42]: They aren't required, so I wouldn't worry about it.

File: platforms/android/platform_www/plugins/com.localytics.phonegap.LocalyticsPlugin/www/localytics.js
Comment (abashyam) [inline platforms/android/platform_www/plugins/com.localytics.phonegap.LocalyticsPlugin/www/localytics.js:457]: the Cordova api should be same for android and iOS.

File: platforms/ios/LocalyticsSample/Classes/AppDelegate.m
Comment (abashyam) [inline platforms/ios/LocalyticsSample/Classes/AppDelegate.m:38]: Use weak linking which ensures the same behavior.This won't detect typos.

File: platforms/ios/Podfile.lock
Comment (abashyam) [inline platforms/ios/Podfile.lock:16]: Should be 1.5.1

File: platforms/ios/www/index.html
Comment (abashyam) [inline platforms/ios/www/index.html:171]: Something wrong with these indents.

File: www/js/index.js
Comment (abashyam) [inline www/js/index.js]: Dont like the appKey here. the Old code is right.

**Review by abashyam — CHANGES_REQUESTED** [review]
I will test this on a device once, I get into office and approve it. appKey is the only item of concern.

File: www/js/index.js
Comment (Mriegel) [reply www/js/index.js]: ya just a mistake after testing - will fix

File: platforms/android/platform_www/plugins/com.localytics.phonegap.LocalyticsPlugin/www/localytics.js
Comment (Mriegel) [reply platforms/android/platform_www/plugins/com.localytics.phonegap.LocalyticsPlugin/www/localytics.js:457]: Not sure I follow... it is the same for Android and iOS (it used to only exist on iOS)

**Comment by kbataran** [comment]
I still think there's some work that could be done with the README... making the instructions for integration clearer to developers who are using this repo to build their own cordova apps. The README in the React Native template app might be a good place for inspiration, let me know if you think the README is fine as it is

**Comment by Mriegel** [comment]
I just mentally diff-ed the react-native README and this one.  Not sure I see what you feel is missing.

**Review by kbataran — COMMENTED** [review]
Two things to be addressed before I'll approve:
1) _autoIntegrate_ button/function does not work
2) You need to remove and link the LocalyticsPlugin twice in order to run the sample app

**Comment by Mriegel** [comment]
Fixed the autoIntegrate option on iOS. Should be able to fix the other issue once we release 5.1.  Will hold off on merging until then.

**Pull request merged**
